Library/brew.sh

```diff
@@ -117,10 +117,10 @@ then
   set -- "$@" -v
 fi
 
+HOMEBREW_ARG_COUNT="$#"
 HOMEBREW_COMMAND="$1"
 shift
 case "$HOMEBREW_COMMAND" in
-  '')          HOMEBREW_COMMAND="help";;
```

Comment:
@xu-cheng Replacing the current fix with `brew help; exit 1` would be an improvement, but IMO only a minor improvement. That would still show the help for these (admittedly silly and unlikely) invocations:

- `brew ''`
- `brew '' --some-other-random-argument`